addon/pods/components/rui-error-template/component.js

```diff
@@ -5,6 +5,12 @@ export default Ember.Component.extend({
   classNames: ['rui-error-template'],
   layout,
 
+  externalLink: Ember.computed(function() {
+    let link = this.get('link')
+
+    return link.indexOf('http') >= 0 ? true : false
```

Comment:
Cool. You could do this here if you wanted (I think that's the right regex).
```
const {  computed } = Ember
externalLink:  computed.match('link',  /^(http|https):///)
```

Other than that looks good!